playbooks/roles/install-pip/tasks/pip.yml

```diff
@@ -0,0 +1,21 @@
+---
+
+- name: Check Python pip exists
+  shell: /usr/bin/which pip &>/dev/null
```

Comment:
No guarantees what /bin/sh aliases to so POSIX compliance required
1. &> should be explicit redirect of fd 1 & 2 to /dev/null
2. "which" is not a shell builtin. In Bourne shells both "which" and "type -P" interrogate PATH - depending on shell, functions and aliases are not evaluated. Use POSIX standard "command -v" instead.